pkgs/applications/misc/diffuse/default.nix

```diff
@@ -57,6 +58,10 @@ python3.pkgs.buildPythonApplication rec {
   # to avoid running gtk-update-icon-cache, update-desktop-database and glib-compile-schemas
   DESTDIR = "/";
 
+  makeWrapperArgs = [
+      "--prefix XDG_DATA_DIRS : ${hicolor-icon-theme}/share"
+  ];
```

Comment:
Does adding `hicolor-icon-theme` to `buildInputs` do the same job?

https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/gnome.section.md#icons-ssec-gnome-icons

Reply:
This is what I initially tried, and it did not work. Adding it to nativeBuildInputs did not work either. It would compile, but a couple of icons on the toolbar would be missing:

`(.diffuse_impl-wrapped:94355): Gtk-WARNING **: 20:21:23.105: Could not find the icon 'new-2way-merge-symbolic-ltr'. The 'hicolor' theme
was not found either, perhaps you need to install it.`

What truly works is to use makeWrapperArgs and change the env variable. I'm not sure whether it's usual. However, I discovered that several other packages in inxpkgs do the same thing.